src/Error/Debugger.php

```diff
@@ -110,6 +111,19 @@ class Debugger
         ],
     ];
 
+    /**
+     * Mapping for error renderers.
+     *
+     * Error renderers are replacing output formatting with
+     * an object based system. Having Debugger handle and render errors
+     * will be deprecated and the new ErrorTrap system should be used instead.
+     *
+     * @array <string, class-string>
+     */
+    protected $renderers = [
+        'txt' => TextRenderer::class,
```

Comment:
`txt` is a little ambiguous. Is this a console output or does the text renderer get used for other things?

Reply:
It is one of the existing output formats that `Debugger` has. This starts the replacement of our default built-in output formats with the new system.

Reply:
Ah, ok. So it will probably be console output, but it's not tied to an IO stream of any kind.

Can we just name it 'text' then?

Reply:
We could. The existing format uses `txt`. We could add `text` as well though.

Reply:
Ok, up to you, if you're trying to support something existing.